### ISSUE TITLE
GPSR

### ISSUE THREAD

**Issue by Krzysztof559** [allegro/allegro-api#10459]
Czy w kategoriach jest jakieś pole które mówi czy dana kategoria jest wykluczona z GPSR?

**Comment by MaciejFrackowiak** [comment]
Nie, w API nie zwracamy żadnej flagi dla wykluczonych kategorii.
